frontend-web/webclient/app/UCloud/FilesApi.tsx

```diff
@@ -861,6 +888,13 @@ async function synchronizationOpOnClick(files: UFile[], cb: ResourceBrowseCallba
     snackbarStore.addSuccess(`${allSynchronized ? "Removed from" : "Added to"} Syncthing`, false);
 }
 
+export function isReadonly(entries: Permission[]): boolean {
+    const isAdmin = entries.includes("ADMIN");
+    const isEdit = entries.includes("EDIT");
```

Comment:
```suggestion
    const isAdmin = entries.includes("ADMIN");
    const isEdit = entries.includes("EDIT") || isAdmin;
```

Reply:
Just as a note, this function has been moved, not changed, so this is how it was before this PR.

Reply:
I noticed that this isn't part of the actual PR. I am pretty sure it should be changed to this though.